wasatch/WasatchDevice.py

```diff
@@ -918,3 +935,25 @@ def change_setting(self, setting, value, allow_immediate=True):
         if (allow_immediate and self.immediate_mode) or re.search(r"trigger|laser", setting):
             log.debug("immediately processing %s", control_object)
             self.process_commands()
+
+    # override handle_requests. I didn't see a good way to overwrite change_setting without
+    # having to redo that pass through.
+    def handle_requests(self, requests: list[SpectrometerRequest]) -> list[SpectrometerResponse]:
+        responses = []
+        for request in requests:
+            try:
+                cmd = request.cmd
+                proc_func = self.process_f.get(cmd, None)
+                if proc_func == None:
+                    try:
+                        self.change_setting(*requests.args, **requests.kwargs)
+                    except:
+                        return []
```

Comment:
I'm having trouble changing integration time or doing anything from ENLIGHTEN.  Couple questions: should that be "request.args" (and .kwargs) instead of "requests"?  And should you log the exception trace here?  Are *you* able to set integration time and so forth?